test/e2e/e2e.go

```diff
@@ -1873,7 +1873,65 @@ var _ = ginkgo.Describe("e2e ingress traffic validation", func() {
 				}
 			}
 		})
+		// This test validates ingress traffic to nodeports with externalTrafficPolicy Set to local.
+		// It creates a nodeport service on both udp and tcp, and creates a backend pod on each node.
+		// The backend pod is using the agnhost - netexec command which replies to commands
+		// with different protocols. We use the "hostname" and "clientip" commands to have each backend
+		// pod to reply with its hostname and the request packet's srcIP.
+		// We use an external container to poke the service exposed on the node and ensure that only the
+		// nodeport on the node with the backend actually receives traffic and that the packet is not
+		// SNATed.
+		// In case of dual stack enabled cluster, we iterate over all the nodes ips and try to hit the
+		// endpoints from both each node's ips.
+		// TODO: Add Test that verifies SNAT=GWRouterIP of ingress traffic does not occur with externalTrafficPolicy=local
```

Comment:
aren't you already doing this with pokeEndpointClientIP ?

Reply:
Yep Erroneous comment I forgot to remove